client/app/Body/Body.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styles from './Body.css';
+import RatingsChart from '../RatingsChart/RatingsChart.jsx';
+
+const Body = ({ attr, rating }) => (
+  <div className={styles.mainContainer}>
+    <div className={styles.infoContainer}>
+      <div className={styles.desc}>{attr.description} NOTE: MAKE THESE LONGER</div>
+      <div className={styles.subContainer}>
+
+        <div className={[styles.column, styles.c1].join(' ')}>
+          <div className={styles.ratingContainer}>
+            <span className={styles.rating}>{rating}</span>
+            <span>*Bubbles*</span>
+            <a className={styles.reviews}>{attr.reviews}</a>
+          </div>
+
+          <RatingsChart />
+
+          <div className={styles.talkAbout}>TRAVELERS TALK ABOUT</div>
```

Comment:
Are you planning to break this component down further into subcomponents in react? Might be easier to reason through later on in the project.

Reply:
I wasn't initially but as I went it definitely started to look like I should. I'll list it as a Technical Debt item on Trello!